slackcli/names.py

```diff
@@ -28,6 +28,14 @@ def username(self, user_id):
             self.user_index[user_id] = slack.client().users.info(user_id).body['user']['name']
         return self.user_index[user_id]
 
+    def user_id(self, username):
+        members = slack.client().users.list().body['members']
```

Comment:
This is not going to work if there are multiple pages of results. This will happen in slack teams with many users.

Reply:
Shouldn't `.users.list()` method return a single list of all users?
https://api.slack.com/methods/users.list